reference-architecture/vmware-ansible/roles/haproxy-server/tasks/main.yaml

```diff
@@ -0,0 +1,36 @@
+---
+- name: Ensure HAProxy is installed.
+  yum: name=haproxy state=installed
+  when: inventory_hostname == 'haproxy-0'
```

Comment:
yea. I build the jinja2 haproxy.cfg template based on infra and master nodes. This conditional ensures that the configuration files and such get passed to it.